plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -36,6 +36,8 @@ type SQLServer struct {
 	HealthMetric bool             `toml:"health_metric"`
 	Log          telegraf.Logger  `toml:"-"`
 
+	UserAssignedID string         `toml:"user_assigned_id"`
```

Comment:
Let's keep the exported variables together. You may need to edit the whole section to ensure the spacing stays consistent.

```suggestion
	HealthMetric   bool             `toml:"health_metric"`
	UserAssignedID string           `toml:"user_assigned_id"`
	Log            telegraf.Logger  `toml:"-"`
```